lib/app/geocode/nominatim.dart

```diff
@@ -127,6 +127,118 @@ class Nominatim {
     }
     return Place.fromJson(data);
   }
+
+  /// Searches a places by their name
```

Comment:
A few of these (countryCode) we could hard-code or force stricter restraints (lon/lat) for the app's specific use-case - not a must though.

Reply:
I assume you mean, what I have already implemented here, or?
https://github.com/verdigado/gruene-app/blob/216_search_on_map/lib/app/services/nominatim_service.dart#L38-L43